test/react/mocks/fb2.js

```diff
@@ -0,0 +1,160 @@
+const React = require("react");
+this['React'] = React;
+
+if (!window.babelHelpers) {
+	window.babelHelpers = {
+		inherits(subClass, superClass) {
+			Object.assign(subClass, superClass);
+			subClass.prototype = Object.create(superClass && superClass.prototype);
+			subClass.prototype.constructor = subClass;
+			subClass.__superConstructor__ = superClass;
+			return superClass;
+		},
+		_extends: Object.assign,
+		extends: Object.assign,
+		objectWithoutProperties(obj, keys) {
+			var target = {};
+			for (var i in obj) {
+				if (!hasOwn.call(obj, i) || keys.indexOf(i) >= 0) {
+					continue;
+				}
+				target[i] = obj[i];
+			}
+			return target;
+		},
+		taggedTemplateLiteralLoose(strings, raw) {
+			strings.raw = raw;
+			return strings;
+		},
+		bind: Function.prototype.bind,
+	};
+}
+
+module.exports =  (function(modules) {
+  // webpackBootstrap
+   // The module cache
+   var installedModules = {}; // The require function
+  
+    function __webpack_require__(moduleId) {
+    
+     // Check if module is in cache
+     if (installedModules[moduleId]) {
+       return installedModules[moduleId].exports;
```

Comment:
Can we use a minimal repro here for clarity? I don't think any of the webpack stuff is necessary.

Reply:
I actually thought it was a good criteria as this is under the mocks tests for fb.

Reply:
Yeah, but we don't actually use webpack at FB in practice so IMO this is obscuring what the test is about (our class transform)